rfc-http-validate.py

```diff
@@ -139,7 +139,7 @@ def location(self):
     def check_start_line(self, start_line):
         if start_line[0].isspace():
             self.validationError(f"Start line starts with whitespace: '{start_line}'")
-            return lines
+            return
         parts = start_line.split(" ")
         if parts[0][-1] == ":":
             return lines  # it must be a header line
```

Comment:
```suggestion
            return  # it must be a header line
```

Sorry, here too -- you've changed the signature to return None